include/sensors/core/tasks/pressure_driver.hpp

```diff
@@ -233,10 +236,15 @@ class MMR920C04 {
             static_cast<uint8_t>(mmr920C04::Registers::PRESSURE_READ),
             static_cast<std::size_t>(3), own_queue,
             static_cast<uint8_t>(mmr920C04::Registers::PRESSURE_READ));
+        if (_sensor_mode == mmr920C04::SensorMode::SINGLE_READ) {
+            writer.write_isr(mmr920C04::ADDRESS,
+                             static_cast<uint8_t>(mmr920C04::Registers::RESET),
+                             data);
+        }
+    }
 
-        writer.write_isr(mmr920C04::ADDRESS,
-                         static_cast<uint8_t>(mmr920C04::Registers::RESET),
-                         data);
+    auto set_sync_bind(SensorOutputBinding binding) -> void {
+        _output_sync_bind = binding;
```

Comment:
we should also call `hardware.reset_sync()` in this function, because
- if this binding contains `::sync`, then we should make sure we start with it reset
- if we were sync-bound, and had raised sync, and this is `::none`, we won't have another opportunity to reset it